examples/stable/mssql/README.md

```diff
@@ -0,0 +1,353 @@
+# Microsoft SQL Server
+
+Microsoft SQL Server is one of the most popular and widely used database in the world.
+
+## Introduction
+This document will cover how to install SQL Server and how to run backup/restore actions.
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.71.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
+## Installing Microsoft SQL Server
+
+### Create Password
+
+```bash
+$ kubectl create secret generic mssql --from-literal=SA_PASSWORD="MyC0m9l&xP@ssw0rd"
+```
+
+### Create storage
+Execute following commands to create PVC for SQL Server installation.
+```bash
+$ cat > pvc.yaml <<EOF
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+     name: azure-disk
+provisioner: kubernetes.io/azure-disk
+parameters:
+  storageaccounttype: Standard_LRS
+  kind: Managed
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mssql-data
+  annotations:
+    volume.beta.kubernetes.io/storage-class: azure-disk
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi
+EOF
+
+$ kubectl create -f pvc.yaml
+```
+
+### Create Deployment
+Create service and deployment by using following code
+
+```bash
+$ cat > deployment.yaml <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mssql-deployment
```

Comment:
it looks like all these resources are being created in `default` namespace. Let's create them  in a specific namespace for example `sqlserver` or something similar.